plexmodule.py

```diff
@@ -208,6 +216,7 @@ def get_watched_shows(shows: List[Show]) -> Optional[List[PlexWatchedSeries]]:
                         show.originalTitle.strip(),
                         year,
                         seasons,
+                        (lambda: series_rating, lambda: seasons_average_rating)[seasons_average_rating > 0](),
```

Comment:
Use `series_rating` here. Add seasons_average_rating as a separate field in `PlexWatchedSeries`.

Also, this syntax is confusing, `seasons_average_rating or series_rating` would have done the same job.

Reply:
Yeah I don't know what I was doing here